### PR TITLE
Revamp QSG

### PR DESCRIPTION
**Pull request by Krishanx92**

### CODE REVIEW

File: en/docs/setup/prerequisites.md [hunk 1]
```diff
@@ -18,3 +18,4 @@
      - Add Jetstack
     
          `helm repo add jetstack https://charts.jetstack.io`
+
```
Comment (AmaliMatharaarachchi) [inline en/docs/setup/prerequisites.md:21]: Shall we add the required helm version here?

Comment (RakhithaRR) [inline en/docs/setup/prerequisites.md:21]: Don't we need to define the minimum helm and k8s versions required?

**Review by sanjeewa-malalgoda — DISMISSED** [review]

File: en/docs/setup/prerequisites.md (hunk 1 above)
Comment (Krishanx92) [reply en/docs/setup/prerequisites.md:21]: Lets add this in separate PR

Comment (Krishanx92) [reply en/docs/setup/prerequisites.md:21]: Lets add this in separate PR

**Review by RakhithaRR — APPROVED** [review]

**Pull request merged**